openapi.yaml

```diff
@@ -647,29 +647,48 @@ components:
       additionalProperties: false
       required:
         - title
-        - summary
+        - startDate
+        - endDate
         - details
         - reward
+        - payoutType
+        - reviewType
         - creatorId
         - workspaceId
         - fields
       properties:
         title:
           type: string
           maxLength: 255
-        summary:
+        startDate:
           type: string
-          maxLength: 1024
+          format: date-time
+          maxLength: 128
+          description: Start date for proposal acceptations
+        endDate:
+          type: string
+          format: date-time
+          description: Deadline for proposal submission
+          maxLength: 128
         details:
           type: string
           maxLength: 4096
-        deadline:
+        reward:
+          $ref: '#/components/schemas/Amount'
+        payoutType:
           type: string
-          format: date-time
-          description: Deadline of the application
           maxLength: 128
-        reward:
-          $ref: '#/components/schemas/GrantReward'
+          enum: [
+            "in one go",
```

Comment:
can we snake case the enums? also the array brackets are unnecessary, better to:
```
enum:
- in-one-go
- milestone
```

Reply:
I was following [this doc ](https://swagger.io/specification/) for `enum`, it uses array brackets, not sure if it works without the brackets

Reply:
yeah it works, enum is just a yaml array -- so idiomatic syntax is what I've written in the previous comment